actionview/lib/action_view/helpers/active_model_helper.rb

```diff
@@ -1,54 +1,9 @@
 # frozen_string_literal: true
 
-require "active_support/core_ext/module/attribute_accessors"
-require "active_support/core_ext/enumerable"
-
 module ActionView
   # = Active Model Helpers
   module Helpers # :nodoc:
     module ActiveModelHelper
     end
-
-    module ActiveModelInstanceTag
-      def object
-        @active_model_object ||= begin
-          object = super
-          object.respond_to?(:to_model) ? object.to_model : object
-        end
-      end
```

Comment:
This method was never called because `ActiveModelInstanceTag` was [included in `Tags::Base`](https://github.com/rails/rails/blob/6e1694275007b94b8112455ac53e8a345938376e/actionview/lib/action_view/helpers/tags/base.rb#L7) rather than prepended.  Calling `to_model` is currently handled by [`ModelNaming#convert_to_model`](https://github.com/rails/rails/blob/6e1694275007b94b8112455ac53e8a345938376e/actionview/lib/action_view/model_naming.rb#L6-L8) instead.